apps/fl/js/utils.js

```diff
@@ -15,6 +25,16 @@ function _(id, args) {
 
 // Show a dialog with one or two buttons
 function showDialog(options) {
+
+  function close(e) {
+    document.body.removeChild(dialog);
+    e.preventDefault();
+    e.stopPropagation();
+  }
```

Comment:
Does jshint really complain about forward references to functions? I prefer to put utility functions like this at the end where they don't clutter things up and obscure the main purpose of the code. It is not worth changing your patch over, but I'm guessing that you were imposing your own style views here, not just making jshint happy...